src/services/AttestationWorkflow.ts

```diff
@@ -111,21 +112,26 @@ class AttestationWorkflow {
     claim: sdk.IClaim,
     attesterAddresses: Array<IContact['publicIdentity']['address']>,
     terms: sdk.AttestedClaim[] = [],
-    delegationId: sdk.IDelegationNode['id'] | null = null,
+    delegationId?: sdk.IDelegationNode['id'],
```

Comment:
Might want to change undefined back to null.

Reply:
I don't think this is possible without changing the `SDK`

Reply:
![Screenshot 2020-07-20 at 09 49 21](https://user-images.githubusercontent.com/48058984/87912854-60f36a80-ca6e-11ea-8839-ec7d64f08fa9.png)

Reply:
Ahh I see, well I guess we should settle on how we want blank properties to behave.